pypeit/core/telluric.py

```diff
@@ -166,16 +216,15 @@ def read_telluric_grid(filename, wave_min=None, wave_max=None, pad_frac=0.10):
 
     Returns:
         :obj:`dict`:  Dictionary containing the telluric grid
-            - wave_grid=
-            - dloglam=
-            - resln_guess=
-            - pix_per_sigma=
-            - tell_pad_pix=
-            - pressure_grid=
-            - temp_grid=
-            - h2o_grid=
-            - airmass_grid=
-            - tell_grid=
+            - wave_grid: Telluric model wavelength grid
+            - dloglam: Wavelength sampling of telluric model
+            - tell_pad_pix: Number of pixels to pad at edges for convolution
+            - pressure_grid: Atmospheric pressure values in telluric grid [mbar]
```

Comment:
I flag this here about the default units used are different from the other units used throughout the code. I don't think this is an issue for this PR, since it's clearly documented, but it might make sense for me to make some changes in different parts of the code to keep the same units as you are using here.